workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -157,7 +157,12 @@ def _set_opts_and_overlay(hmap, overlay="run"):
     return (
         hmap.opts(norm={"framewise": True}, plot=dict(width=850, height=500))
         .overlay(overlay)
-        .opts(legend_position="right")
+        .opts(
+            legend_position="right",
+            bgcolor="lightgray",
```

Comment:
nit: to me the plot would be a little clearer if the background was lighter. Is there something between lightgray and white? Obviously pretty subjective...

Reply:
Apparently 'whitesmoke' is a color.